packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1432,10 +1459,11 @@ export class ContainerRuntime
 	};
 
 	/**
-	 * Summarizer is responsible for coordinating when to send generate and send summaries.
+	 *Summarizer is responsible for coordinating when to send generate and send summaries.
```

Comment:
```suggestion
	 * Summarizer is responsible for coordinating when to send generate and send summaries.
```

Reply:
There seem to be a number of places where the formatting was (unintentionally?) changed to omit the leading space. We should restore those.

Reply:
If this package was using our `recommended` eslint config, it would have caught this. Migration to this config is WIP 🙂